### ISSUE TITLE
On iOS: WebGL: INVALID_FRAMEBUFFER_OPERATION: clear: framebuffer not complete

### ISSUE THREAD

**Issue by naso** [yomboprime/GPGPU-threejs-demos#5]
Hello.

This is likely to be not an issue with the GPUComputationRenderer.js util, but probably some unsupported feature on iOS, but I would like to understand it better if possible.

When testing on iOS I get the following:

> [Warning] WebGL: INVALID_FRAMEBUFFER_OPERATION: clear: framebuffer not complete
> [Warning] WebGL: INVALID_FRAMEBUFFER_OPERATION: drawElements: framebuffer not complete

Do you reckon what can be the cause of it?

Thanks.

**Comment by yomboprime** [comment]
Most probably `float` pixel data in render targets is not supported on your device.  :pensive:

**Comment by yomboprime** [comment]
BTW, these demos are uploaded already to the upstream [three.s repository in github](https://github.com/mrdoob/three.js)

You could ask there if the same demos are working for the people which uses iOS.

**Comment by naso** [comment]
Thanks. And thanks for this helper by the way.

Is it float pixel data the same as the OES_texture_float? Because it passes that OES_texture_float test.

**Comment by naso** [comment]
Ok, I think I got it. Did some research and it seems that in fact **rendering** to floating point textures is the issue on iOS.

https://github.com/pex-gl/pex-glu/issues/3

Half float, seems to work though, and Three.js supports it as I found it here: https://github.com/mrdoob/three.js/pull/5975

So by changing in the renderTarget the texture type to `THREE.HalfFloatType` made it work.

Once again, thanks for you feedback.
